test/integration/VisualizarCards.feature

```diff
@@ -0,0 +1,107 @@
+#language: pt
+Funcionalidade: Visualizar testes dos experimentos como cards
+  Eu como um usuário
+  Eu quero visualizar os testes como cards
+
+Contexto:
+  Dado que estou logado como usuário do ide-experimentador
+  E eu estou na página do experimento
+
+# Cenários Gerais (relacionados a lista de testes)
+# Cenário triste
+Cenário: Não há testes no experimento
+  Dado que nenhum teste do experimento '2' foi criado no sistema
+  Quando eu abrir a página do experimento
+  Então eu deveria ver uma mensagem na tela dizendo "Nenhum teste foi cadastrado no sistema"
+
+# Cenário feliz
+Cenário: Há testes no experimento
+  Dado que o experimento '2' foi criado no sistema
+  E que existem '4' testes não executados no experimento
+  Então eu deveria ver todos os '4' testes não executados como cards
+  E eu deveria ver no primeiro card 'Teste 1', 'x:0 y:0 z:0', bateria: '90%', resultado: 'sucesso', runs:'100'
+  E eu deveria ver no segundo card 'Teste 2', 'x:1 y:1 z:1', bateria: '50%', resultado: 'falha', runs:'1'
+  E eu deveria ver no terceiro card 'Teste 3', 'x:1 y:1 z:1', bateria: '50%', resultado: 'falha', runs:'51'
+  E eu deveria ver no quarto card 'Teste 4', 'x:1 y:1 z:0', bateria: '70%', resultado: 'sucesso', runs:'76'
+  
+# Cenário feliz
+Cenário: Todos os testes executados
+  Dado que todos os '4' testes estão selecionados
+  E que pressionei o botão executar 'experimento 2' no menu lateral
+  Então eu deveria ver ícone do resultado do teste em execução em todos os '4' testes selecionados
+  Quando todos os testes terminarem
+  Então eu deveria ver no primeiro card 'Teste 1', 'x:0 y:1 z:1', bateria: '60%', resultado: 'sucesso', runs:'1'
+  E eu deveria ver no segundo card 'Teste 2', 'x:0 y:0 z:0', bateria: '90%', resultado: 'sucesso', runs:'101'
+  E eu deveria ver no terceiro card 'Teste 3', 'x:1 y:1 z:1', bateria: '50%', resultado: 'falha', runs:'51'
+  E eu deveria ver no segundo card 'Teste 4', 'x:1 y:1 z:0', bateria: '70%', resultado: 'sucesso', runs:'76'
+
+# Cenário feliz
+Cenário: Alguns testes executados
+  Dado que os testes 'Teste 1' e 'Teste 3' estão selecionados
+  E que os testes 'Teste 2' e 'Teste 4' não estão selecionados 
+  E que pressionei o botão executar no menu lateral
+  Então eu deveria ver ícone do resultado do teste em execução em todos os testes selecionados
+  Quando todos os testes selecionados terminarem
+  Então eu deveria ver no primeiro card 'Teste 1', 'x:0 y:1 z:1', bateria: '60%', resultado: 'sucesso', runs:'1'
+  E eu deveria ver no terceiro card 'Teste 3', 'x:1 y:1 z:1', bateria: '50%', resultado: 'falha', runs:'51'
+  E eu deveria ver o 'Teste 2' como estava antes
+  E eu deveria ver o 'Teste 4' como estava antes
+
+
+# Cenários Gerais (relacionados ao card do teste)
+# Cenário triste

```

Comment:
Esse não é um sad path, visto que é esperado que o card seja mostrado sem os dados de um teste que ainda não foi executado. Pensem no sad path como uma situação que vocês usariam try and catch, por exemplo.